### PR TITLE
Update willdurand/negotiation requirement for PHP 8 compatibility

### PR DESCRIPTION
**Pull request by nicolengsy**
The [negotiation package](https://github.com/willdurand/Negotiation/releases/tag/3.0.0) version needs to be ^3.0 in order to be PHP 8 compatible.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     "require": {
         "php": "^7.2",
         "psr/http-message": "^1.0",
-        "willdurand/negotiation": "^2.0"
+        "willdurand/negotiation": "^3.0"
```
Comment (mtrajano) [inline composer.json:20]: I would make this `"willdurand/negotiation": "^2.0 || ^3.0"` as it should still be compatible with 2.x if there are no breaking changes

Comment (mtrajano) [reply composer.json:20]: Actually this is fine it was only a naming change and they actually recommend upgrading to 3.0 for php 8 compat: https://github.com/willdurand/Negotiation/releases/tag/3.0.0

**Comment by akrabat** [comment]
Thanks. I'll look this week.

**Review by mtrajano — COMMENTED** [review]
Can also upgrade the travis file to test on 7.4 and 8.0

**Comment by nicolengsy** [comment]
Hi @akrabat! Any updates on this PR?

**Comment by akrabat** [comment]
When I change the `require`  `"php": "^7.2",` to `"php": "^7.2 || ^8.0"`, all the tests fail, so I need to understand why and update appropriately.

**Comment by akrabat** [comment]
Hmm... I think I need the GH action on master for it to be picked up.

**Comment by akrabat** [comment]
Now it works!

**Comment by akrabat** [comment]
@nicolengsy [version 2.0](https://github.com/akrabat/rka-content-type-renderer/releases/tag/2.0) is now released.

**Comment by nicolengsy** [comment]
@akrabat Thank you so much!

**Comment by ingluisjimenez** [comment]
thank you for merging and releasing @akrabat ❤️

**Pull request merged**